classes/email/filters.php

```diff
@@ -41,42 +41,55 @@ public static function mail_attachment_check( $mail, $data, $form){
                     }
                     continue;
                 }
-                if ( isset( $data[ $field_id ] ) ) {
-                    $file = $data[ $field_id ];
-                } else {
-                    $file = Caldera_Forms::get_field_data( $field_id, $form );
-                }
-                if ( is_array( $file ) ) {
-                    foreach ( $file as $a_file ) {
-                        $file = str_replace( $dir[ 'baseurl' ], $dir[ 'basedir' ], $file );
-                        if ( is_string( $a_file ) && file_exists( $a_file ) ) {
-                            $mail[ 'attachments' ][] = $a_file;
-                        }
-                    }
-                    continue;
-                } else {
-                    $file = str_replace( $dir[ 'baseurl' ], $dir[ 'basedir' ], $file );
-                    if ( is_string( $file ) ) {
-						$files = explode(',', $file);
-						foreach($files as $attachment){
-							$attachment = ltrim( $attachment, " ");
-							if( file_exists($attachment) ){
-								$mail[ 'attachments' ][] = $attachment;
+
+                if( Caldera_Forms_Field_Util::is_cf2_field_type(Caldera_Forms_Field_Util::get_type($field,$form))){
```

Comment:
@New0 I did not want to mess with other file field types so I did all of this in an if/else, but I feel like the changes on L47 also should go on L67, and I don't like the copypasta. But it works.